packages/nodes-base/nodes/Airtop/actions/file/getMany.operation.ts

```diff
@@ -0,0 +1,95 @@
+import {
+	type IExecuteFunctions,
+	type INodeExecutionData,
+	type INodeProperties,
+} from 'n8n-workflow';
+
+import { requestAllFiles } from './helpers';
+import { wrapData } from '../../../../utils/utilities';
+import { apiRequest } from '../../transport';
+import type { IAirtopResponse } from '../../transport/types';
+
+const displayOptions = {
+	show: {
+		resource: ['file'],
+		operation: ['getMany'],
+	},
+};
+
+export const description: INodeProperties[] = [
+	{
+		displayName: 'Return All',
+		name: 'returnAll',
+		type: 'boolean',
+		default: false,
+		description: 'Whether to return all results or only up to a given limit',
+		displayOptions,
+	},
+	{
+		displayName: 'Limit',
+		name: 'limit',
+		type: 'number',
+		displayOptions: {
+			show: {
+				resource: ['file'],
+				operation: ['getMany'],
+				returnAll: [false],
+			},
+		},
+		typeOptions: {
+			minValue: 1,
+			maxValue: 100,
+		},
+		default: 10,
+		description: 'Max number of results to return',
+	},
+	{
+		displayName: 'Session IDs',
+		name: 'sessionIds',
+		type: 'string',
+		default: '',
+		description:
+			'Comma-separated list of <a href="https://docs.airtop.ai/api-reference/airtop-api/sessions/create" target="_blank">Session IDs</a> to filter files by',
+		placeholder: 'e.g. 980875c4-e12d,a302-f3dd51e',
```

Comment:
Might be more clear to show this will be a uuid.
```suggestion
		placeholder: 'e.g. 6aac6f73-bd89-4a76-ab32-5a6c422e8b0b, a13c6f73-bd89-4a76-ab32-5a6c422e8224',
```